plugins/user_quota/web_client/js/userQuota.js

```diff
@@ -334,11 +346,11 @@ girder.userQuota = {
      *                    for none. */
     valueAndUnitsToSize: function (sizeValue, sizeUnits) {
```

Comment:
Could we get rid of this function? If we changed the select element to look like the following:

```
select.form-control.input-sm.g-sizeUnits(model="#{resource.model}")
        option(value='1', selected=(resource.sizeUnits === 0)) bytes
        option(value='1024', selected=(resource.sizeUnits === 1)) kB
        option(value='#{1024*1024}', selected=(resource.sizeUnits === 2)) MB
        option(value='#{1024*1024*1024}', selected=(resource.sizeUnits === 3)) GB
        option(value='#{1024*1024*1024*1024}', selected=(resource.sizeUnits === 4)) TB
```

Then it's a simple matter of doing parseInt() and multiplying.